didcomm/unpack.py

```diff
@@ -51,6 +52,13 @@ async def unpack(
         anonymous_sender=False,
     )
 
+    if is_forward(msg_as_dict) and unpack_config.unwrap_re_wrapping_forward:
```

Comment:
Please remove that method from here. 
1) I think we don't need it.
2) Let's keep all forward-related logic in protocols/forward as much as possible